core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/datastore/events/CreateReviewRequest.java

```diff
@@ -54,4 +56,9 @@ public Review execute(RepositoryContext context) {
 		}
 	}
 	
+	@Override
+	public Permission getPermission() {
+		return new Permission(Permission.BROWSE, Permission.ALL);
```

Comment:
⚠️ This is a "write" request, yet it only requires `Permission.BROWSE` permissions.